samples/customABSample/src/main/java/com/inmobi/customabsample/InterstitialCustomABActivity.java

```diff
@@ -66,6 +66,7 @@ private void setupInterstitial() {
                     public void onAdLoadSucceeded(@NonNull InMobiInterstitial inMobiInterstitial,
                                                   @NonNull AdMetaInfo adMetaInfo) {
                         Log.d(TAG, "onAdLoadSuccessful");
+                        Log.d(TAG, "Bid received: " + adMetaInfo.getBid());
```

Comment:
IMO, The log should be something like "ad loaded with bid"
@SwagatoMondal @manoj897